wikimedia/wikipedia-ios-components
# Add WKEmptyView

**Phabricator:** https://phabricator.wikimedia.org/T342552

### Notes
* This PR Adds an Empty View component to the project and adds it to the watchlist feature
* The Empty View Controller was mainly added to allow for the demo testing

Test Steps:
* Run the demo app and click the empty view button. Make sure they're following the designs on figma and the phab ticket
